pocs/utils/google/README.md

```diff
@@ -0,0 +1,51 @@
+# PANOPTES Google Network
```

Comment:
I'm not keen on the name here, probably because I know too much. How about "Enabling Access to Project PANOPTES Cloud Services" or some such?